src/microbe_stage/OrganelleTemplate.cs

```diff
@@ -19,6 +19,8 @@ public OrganelleTemplate(OrganelleDefinition definition, Hex location, int rotat
         Orientation = rotation;
     }
 
+    public bool PlacedThisSession { get; set; }
```

Comment:
I would like to have a documentation comment on this explaining why this is here and what's it used for.